src/content/blog/tailscale-router-ddev-addon.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Tailscale for DDEV: Simple and Secure Project Sharing"
+pubDate: 2025-08-26
+summary: "Tired of temporary sharing links? Learn how to use the ddev-tailscale-router add-on to get a stable, secure, and private URL for your DDEV projects."
+author: Ajith Thampi Joseph
+featureImage:
+  src: /img/blog/2025/08/ddev_tailscale_flowchart.svg
+  alt: "DDEV Tailscale Router Cover Image"
+  caption: "The ddev-tailscale-router add-on enables cross-device testing, stable webhook URLs, and team collaboration."
+  credit: "Image by Claude AI :)"
+categories:
+  - Guides
+  - TechNotes
+---
+
+I've found that DDEV's [`ddev share`](https://docs.ddev.com/en/stable/users/topics/sharing/) command is a great way to quickly share my local development environment. However, since it uses ngrok, it generates a new, random URL every time, which can be a hassle. For a more permanent and secure solution, I've created the [`ddev-tailscale-router`](https://github.com/atj4me/ddev-tailscale-router) add-on.
+
+This add-on uses [Tailscale](https://tailscale.com/), a service that creates a private and secure network between your devices. It is free for personal use!
+
+As a result, you get a stable, human-readable URL for each of your DDEV projects, which you can access from any device on your Tailscale network.
+
+I've found this approach to be particularly useful for:
+
+- **Cross-device testing:** I can easily test my sites on my phone or tablet without being on the same Wi-Fi network.
+- **Stable webhook URLs:** I can use the permanent Tailscale URL as a reliable endpoint for webhooks, such as those from payment gateways.
+- **Team collaboration:** I can share my development environment with team members so they can see my work in progress.
+
+### How it Works
+
+The `ddev-tailscale-router` add-on works by running a Tailscale container alongside your DDEV project. This container automatically connects to your Tailscale network and securely proxies requests to your project's web container.
+
+### Installation
+
+To get started, follow these steps:
```

Comment:
It makes sense to add a warning about Apple Silicone from https://github.com/atj4me/ddev-tailscale-router#getting-started

We have the majority of users on macOS https://ddev.com/blog/stats-on-ddev-usage-nov-2024/#ddevenvironments

It's important to remember where it can be used.